### DISCUSSION TITLE
Support Multiple Taxes

### DISCUSSION THREAD

**Discussion by mathamel** [duncanmcclean/simple-commerce#1144] [Feature Requests]
We have some use case where 2 taxes apply to a taxe zone
- Canada / Quebec : TPS 5% and TVQ 9.975% 

They both need to be shown on invoice and shown separately, they cannot be merged. 

Product 10.00$
TPS (5%) : 0.50$
TVQ(9.975%) : 1.00$ (0.9975$ round up)
Total:  11.50$ (round up 11.4975$)

Currently, tax only select 1 that fit the product taxe category + taxe zone;

**Comment by duncanmcclean** [comment]
Apologies for the delayed reply!

This isn't currently possible - but it will be in the [next version of Simple Commerce](https://duncanmcclean.com/future-of-simple-commerce).
